src/python-common/ceph/deployment/service_spec.py

```diff
@@ -424,3 +425,38 @@ def __init__(self,
         self.rgw_realm = rgw_realm
         self.rgw_zone = rgw_zone
         self.rgw_frontend_port = rgw_frontend_port
+
+
+class IscsiServiceSpec(ServiceSpec):
+    def __init__(self, service_id, pool=None,
+                 placement=None,
+                 trusted_ip_list=None,
+                 fqdn_enabled=None,
+                 api_port=None,
+                 api_user=None,
+                 api_password=None,
+                 api_secure=None,
+                 ssl_cert=None,
+                 ssl_key=None,
```

Comment:
Not sure we need _all_ these, but as a starting point put all the items the config expects.